src/core_plugins/kibana/index.js

```diff
@@ -53,9 +53,11 @@ module.exports = function (kibana) {
           const configuredUrl = server.config().get('tilemap.url');
           const isOverridden = typeof configuredUrl === 'string' && configuredUrl !== '';
           const tilemapConfig = serverConfig.get('tilemap');
+          const vectormapsConfig = (serverConfig.get('vectormap')) ? serverConfig.get('vectormap') : { layers: [] };
```

Comment:
we would like users to add other data-layers. So they will have the hardcoded layers available, plus the ones that they added themselves.